requirements-dev.in

```diff
@@ -1,16 +1,16 @@
 codecov~=2.1.13
-coverage~=7.2.7
-flake8~=3.9.0
+coverage~=7.3.2
+flake8~=6.1.0
 flake8-black~=0.3.3
-flake8-isort~= 4.1.1
-black~=22.6.0
-isort~= 5.10.1
+flake8-isort~= 6.1.0
+black~=23.10.0
```

Comment:
we should probably do this like botocore and match their version, see: https://github.com/boto/botocore/blob/develop/.pre-commit-config.yaml#L37

Reply:
I will look into that!

Reply:
I added pre-commit as a dev dependency, aligned the pre-commit config with upstream and did a rebase. Do you want me to remove flake8, black and isort from dev dependencies and point everyone to pre-commit?